packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -898,6 +904,7 @@ export const prepareVaultManagerKit = (
               shortfall: shortfallToReserve,
             });
           }
+          liquidateAll();
```

Comment:
If it doesn't break anything please move this next to `reinstateAll`.

Reply:
I had intended to do that. BUT `reinstateAll` is only used on one branch, whereas the `liquidateAll` is outside the big `if` and so applies to all branches.